libraries/engage/product/selectors/product.js

```diff
@@ -107,3 +108,13 @@ export const makeIsBaseProductActive = () => createSelector(
     return baseProduct.active || false;
   }
 );
+
+/**
+ * Creates a selector to get the property of a product based on a given label
+ * @param {string} propertyLabel the label of product coming from the config
+ * @returns {Function}
+ */
+export const getCurrentProductPropertyByLabel = propertyLabel => createSelector(
```

Comment:
you don't need to pass propertyLabel here as param because you have already access to all props inside a selector. so you also have props.widgetSettings.propertyLabel inside here